java/cdk-environment/.gitignore

```diff
@@ -0,0 +1 @@
+BuildEC2.pem
```

Comment:
I think it would be cleaner to put this in the top level .gitignore?

Reply:
What would you put there? I left it here because it's a specific file name that doesn't really make sense out of context.

Reply:
I would just put *.pem. That's a good general exclusion to have anyway.

Reply:
Done